composer.lock

```diff
@@ -11304,5 +11315,5 @@
         "lib-libxml": "*"
     },
     "platform-dev": [],
-    "plugin-api-version": "2.0.0"
+    "plugin-api-version": "2.1.0"
```

Comment:
@anzin could you please generate it using 2.0.0 composer version?

Reply:
Hello @eduard13, thanks for your review, I corrected version 'plugin-api-version' to 2.0.0